docs/generating-code-from-schema.md

```diff
@@ -201,38 +198,27 @@ type TickEdge {
 }
 ```
 
-### generateClientv2
-
-There is a new API for generating client API. To turn on the new version, use the ```generateClientv2``` Gradle configuration option. Note that the v1 API will be deprecated soon. 
-
-This new version relies on the use of generics and solves:
-1) Not being able to handle cycles in the schema, and
-2) Not being able to generate on larger schemas due to too many classes getting generated and out of memory errors. We only generate one class per type in the new implementation.
-
-The projection root needs to be instantiated differently for the v2 API.
-
-v1 API:
-```java
-String query = new GraphQLQueryRequest(
-        new MoviesGraphQLQuery(),
-        new MoviesProjectionRoot().movieId()).serialize();
-```
-
-v2 API:
-```java
-String query = new GraphQLQueryRequest(
-        new MoviesGraphQLQuery(),
-        new MoviesProjectionRoot<>().movieId()).serialize();
-```
-
 Kotlin Projects: 
-Kotlin does not support the Java diamond operator (<>) for inferring type arguments. Instead, pass in [Nothing](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-nothing.html) as both arguments.
+Kotlin does not support the Java diamond operator (<>) for inferring type arguments. Instead, pass in [Nothing](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-nothing.html) as both arguments, as shown in the example:
```

Comment:
cc @srinivasankavitha should we bury or reduce how much we advertise Kotlin support?

Reply:
This is ok I think..it's OSS docs, so there will be kotlin users out there